### PR TITLE
DEV-4311: Impact Counts (sans modal)

### PR DESCRIPTION
**Pull request by dpb-bah**
**High level description:**

Adding the impact counts for the Active DABS Dashboard

**Technical details:**

N/A

**Link to JIRA Ticket:**

[DEV-4311](https://federal-spending-transparency.atlassian.net/browse/DEV-4311)

**Mockup**
https://bahdigital.invisionapp.com/share/KAIACS64MV9#/screens/296030449

The following are ALL required for the PR to be merged:

Author: 
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled Demo including Design/Testing/Front-end OR Provided Instructions for Local Testing above and in JIRA
- [x] Verified cross-browser compatibility
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [x] Design review completed
- [x] Frontend review completed
- [x] Merged after #1309 to make sure the headers are the same

### CODE REVIEW

File: src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                this.setState({
+                    hasFailed: false,
+                    submissionData: data,
+                    inFlight: false
+                });
+            })
+            .catch((err) => {
+                console.error(err);
+                this.setState({
+                    hasFailed: true,
```
Comment (ebdabbs) [inline src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx:61]: Looks like we have [DEV-4320](https://federal-spending-transparency.atlassian.net/browse/DEV-4320) to ensure error/loading/no results states, but we should probably check with @AGuyNamedMarco ahead of time to see what he thinks that would look like for this section.

File: src/js/components/dashboard/impacts/ImpactGauge.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/**
+ * ImpactGauge.jsx
+ * Created by Daniel Boos 4/1/2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    submissionData: PropTypes.object,
```
[the remark this answers is not in the document]
Comment (dpb-bah) [reply src/js/components/dashboard/impacts/ImpactGauge.jsx:10]: I can do that. Is it expected to go all the way down if they're nested objects?

File: src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx (hunk 1 above)
Comment (dpb-bah) [reply src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx:61]: Including JIRA ticket to address this: https://federal-spending-transparency.atlassian.net/browse/DEV-4811

File: src/js/components/dashboard/impacts/ImpactGauge.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ebdabbs) [reply src/js/components/dashboard/impacts/ImpactGauge.jsx]: They can get pretty big, but ideally yes. USAS has a [directory for just for PropTypes](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/propTypes/index.js) so they can be imported / reused.

File: src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx (hunk 1 above)
Comment (ebdabbs) [reply src/js/containers/dashboard/ActiveDashboardImpactsContainer.jsx:61]: Thanks!

**Pull request merged**